test/spec/ol/interaction/modifyinteraction.test.js

```diff
@@ -106,7 +106,6 @@ describe('ol.interaction.Modify', function() {
       expect(feature.getGeometry().getRevision()).to.equal(1);
       expect(feature.getGeometry().getCoordinates()[0]).to.have.length(5);
 
-      simulateEvent('pointermove', 10, -20, false, 0);
```

Comment:
This makes me suspicious. These tests should still pass, without the need to modify the event sequence.